workitem/link/type_repository.go

```diff
@@ -65,12 +69,22 @@ func (r *GormWorkItemLinkTypeRepository) Create(ctx context.Context, name string
 	if db.Error != nil {
 		return nil, errors.NewInternalError(fmt.Sprintf("Failed to find work item link category: %s", db.Error.Error()))
 	}
+	// Check space exists
+	space := space.Space{}
+	db = r.db.Where("id=?", linkType.SpaceID).Find(&space)
+	if db.RecordNotFound() {
+		return nil, errors.NewBadParameterError("work item link space", linkType.SpaceID)
+	}
+	if db.Error != nil {
+		return nil, errors.NewInternalError(fmt.Sprintf("Failed to find work item link space: %s", db.Error.Error()))
+	}
+
 	db = r.db.Create(linkType)
 	if db.Error != nil {
 		return nil, errors.NewInternalError(db.Error.Error())
 	}
 	// Convert the created link type entry into a JSONAPI response
-	result := ConvertLinkTypeFromModel(*linkType)
+	result := ConvertLinkTypeFromModel(goa.ContextRequest(ctx), *linkType)
```

Comment:
@aslakknutsen I don't understand why we need it. But maybe it is just too late... zzzZZZ